zkvm/src/prover.rs

```diff
@@ -34,14 +34,16 @@ impl<'a, 'b> Delegate<r1cs::Prover<'a, 'b>> for Prover<'a, 'b> {
             Commitment::Open(w) => (w.value.into(), w.blinding),
             Commitment::Closed(_) => return Err(VMError::WitnessMissing),
         };
-        Ok(self.cs.commit(v, v_blinding))
+        let var = self.cs.commit(v, v_blinding).1;
+        Ok((com.to_point(), var))
```

Comment:
what's the motivation for using `com.to_point()` instead of the first value of `self.cs.commit(v, v_blinding)`?

Reply:
ah, good catch! this is left over from debugging the blinding factor business :)